tests/layer_tests/pytorch_tests/test_matmul.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import pytest
+import torch
+from pytorch_layer_test_class import PytorchLayerTest
+
+class TestMatMulOperation(PytorchLayerTest):
+    def _prepare_input(self, matrix, vector):
+        matrix_input = torch.tensor(matrix, dtype=torch.float32)
+        vector_input = torch.tensor(vector, dtype=torch.float32)
+        return [matrix_input, vector_input]
+
+    def create_model(self, matrix, vector):
+        class CustomMatMulOperation(torch.nn.Module):
+            def forward(self, matrix, vector):
+                return torch.matmul(matrix, vector)
+
+        model_class = CustomMatMulOperation()
+        ref_net = None
+        return model_class, ref_net, "aten::mm"
```

Comment:
this test does not cover the operation you are supporting, it tests that there is aten::mm in the graph instead of aten::mv.
For having your operation in model you need to call torch.mv instead of torch.matmul